### DISCUSSION TITLE
Game stops during break Android

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#25102] [Q&A]
When playing a map when a break starts the game just stops no music the background bim dose not change and time left does stays at 3:01 I can still tap and move the cursor but that's it
![Screenshot_20231012-133453.png](https://github.com/ppy/osu/assets/120297366/637eaeed-ec3c-4a09-9f8f-a03a5be82fff)

Logs
No logs where generated 
![Screenshot_20231012-135501_My Files.png](https://github.com/ppy/osu/assets/120297366/bd2b0ff8-79b3-4ad0-8db9-e2f46f7e1ba6)

**Comment by peppy** [comment]
Wouldn't be surprised if this is something mono runtime level.
